bitcoin/src/blockdata/block.rs

```diff
@@ -582,7 +584,7 @@ mod tests {
             real_decode.block_hash()
         );
         assert_eq!(real_decode.header.difficulty(&params), 2456598);
-        assert_eq!(real_decode.header.difficulty_float(), 2456598.4399242126);
+        // assert_eq!(real_decode.header.difficulty_float(&params), 2456598.4399242126);
```

Comment:
If you're going to remove it. delete it.

But why are you deleting it?

Reply:
Oh, I just forgot to uncomment it.